### PR TITLE
Python 3.12 rebuild

### PR DESCRIPTION
**Pull request by matthiasdiener**
Followup of #30.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by matthiasdiener** [comment]
Error message:

```
ERROR: Python version is incompatible. Found 3.12, which is not yet supported for package builds.
```

File: recipe/ver.diff [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/SConstruct b/SConstruct
+index f7d77a273..8320f4224 100644
+--- a/SConstruct
++++ b/SConstruct
+@@ -1732,7 +1732,7 @@ env['python_cmd_esc'] = quoted(env['python_cmd'])
+ # Python Package Settings
+ python_min_version = parse_version("3.8")
+ # Newest Python version not supported/tested by Cantera
+-python_max_p1_version = parse_version("3.12")
++python_max_p1_version = parse_version("3.13")
```
[the remark this answers is not in the document]
Comment (bryanwweber) [reply recipe/ver.diff:10]: Yeah feel free, although I doubt we'll backport it and release 3.0.1 so this patch will be needed until 3.1 comes out

[the remark this answers is not in the document]
Comment (matthiasdiener) [reply recipe/ver.diff:10]: Thanks, I added this patch here https://github.com/Cantera/cantera/pull/1648.

**Review by bryanwweber — APPROVED** [review]
Lgtm thanks!

**Pull request merged**